### DISCUSSION TITLE
(Game) GUILTY GEAR Xrd -REVELATOR- | Can't unpack Ramlethal's files

### DISCUSSION THREAD

**Discussion by SakuraUO** [gildor2/UEViewer#268] [Q&A] [answered]
1.) After using UEViewer once I get an error saying something along the lines of "Probably file is encrypted."

2.) So, after decrypting them I get this error message...

RawArray item size mismatch: expected 40, serialized 36

SerializeBulkArray <- FSkeletalMeshVertexBuffer3<< <- FStaticLODModel3<< <- TArray::Serialize: 0/1 <- USkeletalMesh3::Serialize <- LoadObject: SkeletalMesh3'ADV_AVT_RAM_SF_dec.ram_body_bandage', pos=6FBC2F, ver=868/3, game=ue3 <- UObject::EndLoad <- LoadWholePackage: ADV_AVT_RAM_SF_dec.upk <- CUmodelApp::ShowPackageUI <- Main: umodel_build=1586

**Comment by gildor2** [comment, marked as answer]
This game has special override which you're not using. If you have any problems with it, check the forum first, please - there are probably some limitations, and things were already explained in several threads.

**Comment by SakuraUO** [comment]
Found them. Got it!

https://www.gildor.org/smf/index.php/topic,2561.0.html

https://www.gildor.org/smf/index.php/topic,2903.0.html
